app/assets/stylesheets/components/news_signup_inpage/_news_signup_inpage.scss

```diff
@@ -0,0 +1,160 @@
+$bg-img-stripe: url(data:image/svg+xml,%3Csvg%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20width%3D%228%22%20height%3D%228%22%20viewBox%3D%220%200%2080%2080%22%20enable-background%3D%22new%200%200%2080%2080%22%3E%3Cpath%20stroke%3D%22%23999%22%20d%3D%22M0%2080L80%200%22%2F%3E%3C%2Fsvg%3E);
+
+%stripe-bg {
+  background-image: $bg-img-stripe;
+}
+
+.news-signup-inpage {
+  @extend %stripe-bg;
+  @include clearfix;
+  position: relative;
+  background-color: $concrete;
+  border-radius: $border-radius;
+  width:100%;
+  padding-bottom:$baseline-unit*2;
```

Comment:
Very small thing, but could you put spaces after the colon, like this... ```property-name: value```

Could be worth checking all your scss files here.